### PR TITLE
New section on ICMP

### PR DESCRIPTION
**Pull request by mirjak**
addresses #200

### CODE REVIEW

File: draft-ietf-quic-manageability.md [hunk 1]
```diff
@@ -898,6 +898,19 @@ are likely to fail, and are not recommended to use as a way to construe
 internals of QUIC's operation as those mechanisms can change, e.g., due to the
 use of extensions.
 
+## Handling ICMP Messages
+
+Datagram Packetization Layer PMTU Discovery (PLPMTUD) can be used by QUIC to
+probe for the supported PMTU. PLPMTUD optionally use ICMP messages (e.g.,
```
Comment (gorryfair) [inline draft-ietf-quic-manageability.md:904]: /optionally use/can optionally use/ or /optionally uses/

Comment (mirjak) [inline draft-ietf-quic-manageability.md:904]:
```suggestion
probe for the supported PMTU. PLPMTUD optionally uses ICMP messages (e.g.,
```

File: draft-ietf-quic-manageability.md [hunk 2]
```diff
@@ -898,6 +898,19 @@ are likely to fail, and are not recommended to use as a way to construe
 internals of QUIC's operation as those mechanisms can change, e.g., due to the
 use of extensions.
 
+## Handling ICMP Messages
+
+Datagram Packetization Layer PMTU Discovery (PLPMTUD) can be used by QUIC to
+probe for the supported PMTU. PLPMTUD optionally uses ICMP messages (e.g.,
+IPv6 Packet Too Big messages). Given know attacks with the use of ICMP messages,
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md:905]:
```suggestion
IPv6 Packet Too Big messages). Given known attacks with the use of ICMP messages,
```

File: draft-ietf-quic-manageability.md [hunk 3]
```diff
@@ -898,6 +898,19 @@ are likely to fail, and are not recommended to use as a way to construe
...
+the use of PLPMTUD in QUIC has been designed to safely utilise received ICMP
+but do not rely upon them to function (see section 14.2.1. of
+{{QUIC-TRANSPORT}}).
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md:908]:
```suggestion
the use of PLPMTUD in QUIC has been designed to safely use but not rely on
receiving ICMP feedback (see {{Section 14.2.1. of QUIC-TRANSPORT}}).
```

File: draft-ietf-quic-manageability.md [hunk 4]
```diff
@@ -898,6 +898,19 @@ are likely to fail, and are not recommended to use as a way to construe
...
+
+Networks are recommended to forward these ICMP messages and retain as much of
+the original packet as possible without exceeding the minimum MTU for the IP
+version when generating ICMP messages as recommended in {{(RFC1812, RFC4443)}}.
```
Comment (martinthomson) [inline draft-ietf-quic-manageability.md:912]: Wow, I didn't realize that this syntax was even possible.  I might have to use that in future.

Comment (mirjak) [reply draft-ietf-quic-manageability.md:912]: Actually is not. The build is broken and I didn't check that yesterday. But would be nice...

File: draft-ietf-quic-manageability.md [hunk 5]
```diff
@@ -898,6 +898,18 @@ are likely to fail, and are not recommended to use as a way to construe
...
+
+Datagram Packetization Layer PMTU Discovery (PLPMTUD) can be used by QUIC to
+probe for the supported PMTU. PLPMTUD optionally uses ICMP messages (e.g.,
+IPv6 Packet Too Big messages). Given known attacks with the use of ICMP messages,
+the use of PLPMTUD in QUIC has been designed to safely use but not rely on
+receiving ICMP feedback (see {{Section 14.2.1. of QUIC-TRANSPORT}}).
+
+Networks are recommended to forward these ICMP messages and retain as much of
+the original packet as possible without exceeding the minimum MTU for the IP
+version when generating ICMP messages as recommended in {{(RFC1812, RFC4443)}}.
```
Comment (mirjak) [inline draft-ietf-quic-manageability.md:911]:
```suggestion
version when generating ICMP messages as recommended in {{RFC1812, RFC4443}}.
```

File: draft-ietf-quic-manageability.md [hunk 6]
```diff
@@ -898,6 +898,18 @@ are likely to fail, and are not recommended to use as a way to construe
...
+
+Datagram Packetization Layer PMTU Discovery (PLPMTUD) can be used by QUIC to
+probe for the supported PMTU. PLPMTUD optionally uses ICMP messages (e.g.,
+IPv6 Packet Too Big messages). Given known attacks with the use of ICMP messages,
+the use of PLPMTUD in QUIC has been designed to safely use but not rely on
+receiving ICMP feedback (see {{Section 14.2.1. of QUIC-TRANSPORT}}).
+
+Networks are recommended to forward these ICMP messages and retain as much of
+the original packet as possible without exceeding the minimum MTU for the IP
+version when generating ICMP messages as recommended in {{RFC1812, RFC4443}}.
```
Comment (mirjak) [inline draft-ietf-quic-manageability.md:911]:
```suggestion
version when generating ICMP messages as recommended in {{RFC1812}} {{RFC4443}}.
```

File: draft-ietf-quic-manageability.md [hunk 7]
```diff
@@ -898,6 +898,18 @@ are likely to fail, and are not recommended to use as a way to construe
...
+
+Datagram Packetization Layer PMTU Discovery (PLPMTUD) can be used by QUIC to
+probe for the supported PMTU. PLPMTUD optionally uses ICMP messages (e.g.,
+IPv6 Packet Too Big messages). Given known attacks with the use of ICMP messages,
+the use of PLPMTUD in QUIC has been designed to safely use but not rely on
+receiving ICMP feedback (see {{Section 14.2.1. of QUIC-TRANSPORT}}).
+
+Networks are recommended to forward these ICMP messages and retain as much of
+the original packet as possible without exceeding the minimum MTU for the IP
+version when generating ICMP messages as recommended in {{RFC1812}} {{RFC4443}}.
```
Comment (britram) [inline draft-ietf-quic-manageability.md:911]:
```suggestion
version when generating ICMP messages as recommended in {{RFC1812}} and {{RFC4443}}.
```

**Review by britram — APPROVED** [review]

**Pull request merged**